assemblies/upgrading-odh-v2.adoc

```diff
@@ -12,13 +12,15 @@ You can upgrade the Open Data Hub Operator from version 2.0 or 2.1 to version 2.
 For information about upgrading from version 1 to version 2.2 or later, see link:{odhdocshome}/upgrading_open_data_hub/#upgrading-the-odh-operator-v1_upgradev1[Upgrading Open Data Hub version 1 to version 2].
 For information about installing the Open Hub Operator, see link:{odhdocshome}/installing_open_data_hub/#installing-the-odh-operator-v2_installv2[Installing Open Data Hub Operator version 2].
 
+[NOTE]
+--
+After you upgrade to {productname-short} 2.10.0, pipelines created with DSP 1.0 will continue to run, but will be inaccessible from the {productname-short} dashboard. We recommend that current DSP users stay on an earlier version of {productname-short} until you are ready to migrate to the new pipelines solution.
+--
 [IMPORTANT]
 ====
 Data Science Pipelines (DSP) 2.0 contains an installation of Argo Workflows. {productname-short} does not support direct customer usage of this installation of Argo Workflows.
 
-If there is an existing installation of Argo workflows that is not installed by DSP on your cluster, installations of and upgrades to {productname-short} 2.10.0 with DSP will fail.
-
-To install or upgrade to {productname-short} 2.10.0 with DSP, ensure that no separate installation of Argo workflows exists on your cluster.
+If you upgrade to {productname-short} 2.10.0 with DSP enabled and an existing Argo Workflows installation that is not installed by DSP on your cluster, {productname-short} components will not be upgraded. To complete the component upgrade, disable DSP or remove the separate installation of Argo Workflows. The component upgrade will complete automatically.
```

Comment:
Slightly unclear, is it meant to be read as: _If you upgrade to {productname-short} 2.10.0 with DSP enabled and an existing Argo Workflows installation that is not installed by DSP **is** on your cluster, {productname-short} components will not be upgraded_ ?

Suggest: 
_If you upgrade to {productname-short} 2.10.0 with DSP enabled and an Argo Workflows installation that is not installed by DSP exists on your cluster, {productname-short} components will not be upgraded_ 
OR
 _If you upgrade to {productname-short} 2.10.0 with DSP enabled on a cluster with an existing Argo Workflows installation that is not installed by DSP, {productname-short} components will not be upgraded_
OR
{productname-short} components will not be upgraded on a cluster with an existing Argo Workflows installation that is not installed by DSP, if you upgrade to {productname-short} 2.10.0 with DSP enabled.

Reply:
Good shout, updated.